manuscript/references.tex

```diff
@@ -0,0 +1,29 @@
+    \bibitem{sealion_cdr}Academy, O. Critical Design Review: Mission SeaLion - ODU/CGA 3U CubeSat.  (2022)
+    \bibitem{heidt_new}Heidt, H., Puig-Suari, J., Moore, A., Nakasuka, S. \& Twiggs, R. CubeSat - A new generation of picosatellite for education and industry low-cost space experimentation. {\em AIAA/USU Annual Conference On Small Satellites, 12th, Utah State University, Logan; UNITED STATES; 21-24 Aug.2000}. (2000,8), https://www.proquest.com/conference-papers-proceedings/cubesat-new-generation-picosatellite-education/docview/27219077/se-2
+    \bibitem{swartwout_data}Swartwout, M. CubeSat Database.  (2021), https://sites.google.com/a/slu.edu/swartwout/cubesat-database
+    \bibitem{pumpkin_cubesat}Shop, C. Pumpkin CubeSat Kits.  (2023,3), https://www.cubesatshop.com/product/pumpkin-cubesat-kits/
+    \bibitem{cubesat_handbook}Cappelletti Cubesat handbook: From mission design to operations. (Elsevier Science \& Technology,2020)
+    \bibitem{reliving_24}Swartwout, M. Reliving 24 Years in the Next 12 Minutes: A Statistical and Personal History of University-Class Satellites.  (2018), https://digitalcommons.usu.edu/cgi/viewcontent.cgi?article=4277\&context=smallsat
+    \bibitem{aalto}Praks AALTO-1 earth observation cubesat mission — Educational outcomes. {\em IEEE International Geoscience And Remote Sensing Symposium (IGARSS)}. (2015)
+    \bibitem{howtosetup}Berthoud, M. How to Set Up a CubeSat Project - Preliminary Survey Results. {\em 30th Annual AIAA/USU Conference On Small Satellites}. (2016)
+    \bibitem{sealion_mission_architecture}Team, S. SeaLion Mission Architecture. (Old Dominion University,2022), https://github.com/ODU-CGA-CubeSat/sealion-mission-architecture
+    \bibitem{caesar_model_based_approach}David Wagner CAESAR Model-Based Approach to Harness Design. {\em Proceedings Of IEEE Aerospace Conference}. (2020,3)
+    \bibitem{ibm_mbse}Brown, B. Model-based systems engineering: Revolution or evolution?.  (2011,12)
+    \bibitem{docs_as_code}Holscher, E. Docs as Code.  (2022), https://www.writethedocs.org/guide/docs-as-code/
+    \bibitem{mach30_git}Simmons, J. Mach30 Modeling Language. (Mach30 Foundation,2022), https://github.com/Mach30/m30ml
+    \bibitem{sys_ml}Partners, S. SysML Specifications.  (2023), https://sysml.org/sysml-specs/
+    \bibitem{sys_ml2}Submission Team, S. SysML-v2-Release.  (2023), https://github.com/Systems-Modeling/SysML-v2-Release
+    \bibitem{plantuml}PlantUML PlantUML.  (2023), https://plantuml.com/
+    \bibitem{oml_language}Maged Elaasar, N. Ontological Modeling Language: Origin and Rationale.  (2022), http://www.opencaesar.io/oml/
+    \bibitem{oml_origin_and_rationale}Jenkinis, S. Ontological Modeling Language 1.4.  (2022), http://www.opencaesar.io/imce/2021/06/19/OML-Origin-and-Rationale.html
+    \bibitem{sealion_dof}Team, S. Distributed OSHW Framework (DOF). (Old Dominion University,2023), https://odu-cga-cubesat.github.io/dof-cubesat/
```

Comment:
should change `Distributed OSHW Framework (DOF)` to `Extension of the Distributed OSHW Framework (DOF) for modeling CubeSats`